src/python_routing_v02/compute_nhd_routing_SingleSeg_v02.py

```diff
@@ -101,8 +101,16 @@ def _handle_args():
 if ENV_IS_CL:
     root = pathlib.Path("/", "content", "t-route")
 elif not ENV_IS_CL:
+    
     root = pathlib.Path("../..").resolve()
     sys.path.append(r"../python_framework_v02")
+    
+    routing_v02_dir = os.path.join(root, "src", "python_routing_v02","fast_reach")
+    sys.path.append(routing_v02_dir)
+    
+    print(root)
+    print(routing_v02_dir)
+    
 
```

Comment:
It doesn't look like these changes work. Not sure why. I made an updated suggestion that appears to give the path in my recommended edits PR to your branch here. There are a number of additional changes to make in general to the calling structure of the script that we hold for now.

Reply:
looks like the changes you made in the PR on my branch worked out. So long as a valid shared object is created in `/fast_reach` tests work.